daemon/pool.go

```diff
@@ -18,10 +18,12 @@ import (
 	"gopkg.in/src-d/go-errors.v1"
 )
 
-const (
+var (
 	// DefaultPoolTimeout is the time a request to the DriverPool can wait
 	// before getting a driver assigned.
-	DefaultPoolTimeout = 5 * time.Second
+	//
+	// Deprecated: use the context to specify the timeout; default was set to 5 sec
+	DefaultPoolTimeout struct{}
```

Comment:
Can we get rid of this entirely? By changing the type we've already made a breaking change in the package API, so at that point we might as well just drop the name entirely I think.

Reply:
Just wanted to have a piece of docs there to specify the solution. My first thought was to drop it as well, so if it's not an issue, I'll do it.

Reply:
I don't object to dropping it. It would be worth having a note in the release notes for the next SDK release, though.